deploy/helm/internal-charts/search-api-node/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
Probably increase to v2 so we can safely to use helm3 feature / ignore helm2 issues.
Also add `kubeVersion: ">= 1.14.0-0"` will make sure users get proper error message when they deploy Magda to a unsuppported Kube version